### PR TITLE
Portfolio1

### PR DESCRIPTION
**Pull request by Noctiscreo**
New layout.

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+/* navbar padding */
+nav {
+	overflow: auto;
+	padding: 15px;
+}
+
+/* nav container*/
+nav .link {
+	width: 100%;
```
Comment (mporam) [inline portfolio.css:54]: this actually does nothing

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Portfolio</title>
+		<link href="normalize.css" rel="stylesheet" type="text/css">
```
Comment (mporam) [inline portfolio.html:6]: fix indentation here

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+				<a href="#" class="contact">Contact Me</a>
+		</div>
+		
+	</div>
+	</nav>
+	
+	<div class="main">
+		<div class="imagebanner">
+			<div class="skills">
+			<h2>Skillset</h2>
```
Comment (mporam) [inline portfolio.html:42]: fix indentation

File: portfolio.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+			<h3>HTML</h3>
+			<h3>CSS</h3>
+			<h3>PHP</h3>
+			<h3>Javascript</h3>
+			<h3>Scrum</h3>
+			<h3>Agile</h3>
+			<h3>Pending</h3>
+			<h3>Pending</h3>
+			<h3>Pending</h3>
```
Comment (mporam) [inline portfolio.html:51]: missing a closing div

File: portfolio.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+		</div>
+	</div>
+
+
+	
+	<div class="projects">
+	<img src="#" alt="Portfolio One">
```
Comment (mporam) [inline portfolio.html:58]: fix indentation

File: portfolio.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+	<h3>PHP</h3>
+	
+	<h3>Javascript</h3>
+	</div>
+	
+	<footer>
+		&copy; Copyright: John Hasznosi - 2019
+	</footer>
+	
+</body>
```
Comment (mporam) [inline portfolio.html:75]: missing closing html tag

**Review by mporam — CHANGES_REQUESTED** [review]

File: portfolio.css (hunk 1 above)
Comment (Noctiscreo) [reply portfolio.css:54]: Hi Mike. When I delete 'width: 100%' my navbar links buch up and move to the left.

File: portfolio.html (hunk 2 above)
Comment (Noctiscreo) [reply portfolio.html]: <head>
	<meta charset="UTF-8">
	<title>Portfolio</title>
	<link href="normalize.css" rel="stylesheet" type="text/css">

Like this?

Comment (Noctiscreo) [reply portfolio.html]: <head>
	<meta charset="UTF-8">
	<title>Portfolio</title>
	<link href="normalize.css" rel="stylesheet" type="text/css">

Like this?

**Pull request merged**